selfdrive/ui/tests/test_ui/run.py

```diff
@@ -175,6 +180,7 @@ def setup_pair_device(click, pm: PubMaster):
   "pair_device": setup_pair_device,
   "settings_device": setup_settings_device,
   "settings_toggles": setup_settings_toggles,
+  "settings_developer": setup_settings_developer,
```

Comment:
You also need to add it here: https://github.com/commaai/openpilot/blob/9daec14220f6fa8ba95e57a9801d622143c8030f/.github/workflows/ui_preview.yaml#L87

Reply:
Fixed!